src/window.py

```diff
@@ -431,9 +446,9 @@ def translateSet(text):
                 (1, "X"): rotate(1, 0, 0),
                 (1, "Y"): rotate(0, 1, 0),
                 (1, "Z"): rotate(0, 0, 1),
-                (2, "X"): None,
-                (2, "Y"): None,
-                (2, "Z"): None,
+                (2, "X"): scale(1, 0, 0),
+                (2, "Y"): scale(0, 1, 0),
```

Comment:
my point is that for me it would have more meaning if you put here not 1, but 1/100

Reply:
It looks like it won't work. The value 1 and 0 act as a boolean to determine which axis was changed.